src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -872,12 +872,18 @@ private void verifyCheckInOperationRecorded(UUID itemId, UUID servicePoint) {
 
     assertThat(recordedOperations.totalRecords(), is(1));
 
+    final String itemEffectiveLocationId = itemsClient.getById(itemId).getJson()
+      .getString("effectiveLocationId");
+
     recordedOperations.forEach(checkInOperation -> {
       assertThat(checkInOperation.getString("occurredDateTime"),
         withinSecondsBeforeNow(Seconds.seconds(2)));
       assertThat(checkInOperation.getString("itemId"), is(itemId.toString()));
       assertThat(checkInOperation.getString("servicePointId"), is(servicePoint.toString()));
       assertThat(checkInOperation.getString("performedByUserId"), is(getUserId()));
+      assertThat(checkInOperation.getString("itemStatusPriorToCheckIn"), is("Checked out"));
```

Comment:
Please can an example be added for a status other than checked out

Reply:
There are tests in `src/test/java/api/loans/InHouseUseCheckInTest.java` that expects status to be `Available`. Should I added one more test to this file?

Reply:
Apologies, given there are tests in another place that I missed, please disregard this comment